cmd/minikube/cmd/kubectl.go

```diff
@@ -87,3 +126,7 @@ func KubectlCommand(version string, args ...string) (*exec.Cmd, error) {
 
 	return exec.Command(path, args...), nil
 }
+
+func init() {
+	kubectlCmd.Flags().BoolVar(&useSSH, "ssh", false, "Use SSH for running kubernetes client")
```

Comment:
does this mean, it will be using the kubectl inside minikube ? (is there a reason that we dont do that all the time ?)

Reply:
lets clarify the help text


how about add a section either in FAQ https://minikube.sigs.k8s.io/docs/faq/ or the SSH driver page https://minikube.sigs.k8s.io/docs/drivers/ssh/
how to use this.

maybe with a way that when users google it, they find it, "How to use kubectl with SSH driver on minikube ? or how to talk to minikube cluster without kubectl ? "

Reply:
There are lots of reasons why we don't have this as the default, one reason is working with files...
```
  # Apply the configuration in pod.json to a pod.
  kubectl apply -f ./pod.json
```

You cannot do "kubectl apply" unless you use a path that are on both. You need to use `-` and `<`, etc.

 ```
 # Apply the JSON passed into stdin to a pod.
  cat pod.json | kubectl apply -f -
```

Reply:
Added more help text, also noticed that "minikube kubectl -- --help" had also been broken by --cluster